### DISCUSSION TITLE
Issues faced while cloning a project.

### DISCUSSION THREAD

**Discussion by Jslnvarghese** [community/community#46706] [New to GitHub]
https://github.com/Danny1024-na/amazonClone
I forked this project and clone it in my git hub desktop.
But it didn't worked properly.
The files which are showed in the repo is not found when it cloned.
It is very difficult to find the files and understand the code ,similarly I cloned some other projects but  the same thing happened.
Can you please help me out.

**Comment by Juanescacha** [comment]
can you please share screenshots to understand better the issue your having?
what files are missing? have you tried instead downloading it as a simple .zip instead?
maybe you have to show hidden files on your PC or something simliar

**Comment by lbirkert** [comment]
> https://github.com/Danny1024-na/amazonClone I forked this project and clone it in my git hub desktop. But it didn't worked properly. The files which are showed in the repo is not found when it cloned. It is very difficult to find the files and understand the code ,similarly I cloned some other projects but the same thing happened. Can you please help me out.

Consider doing this via the CLI, it is easier.

`git clone https://github.com/Danny1024-na/amazonClone`

**Comment by ghost** [comment]
Save
